agent/src/agent/pipeline/json_builder/observium.py

```diff
@@ -120,6 +120,8 @@ def _load_config(self):
         self.config['dimensions'] = self._dimensions()
         self._validate_dimensions()
         self.config['values'] = self._measurements()
+        self.config['values_configurations'] = self._values_configurations()
```

Comment:
could you explain why do we have value and dimension configurations here but don't have tags?